Add nutrition facts for OIE files (#2560)
okta/okta-developer-docs: packages/@okta/vuepress-site/docs/guides/oie-embedded-widget-use-case-load/main/index.md

<StackSelector class="cleaner-selector"/>

> This guide covers the use case to load the widget, and provides a flow diagram and a sequence of integration steps.

Reviewer: ```suggestion
This guide covers the use case to load the Widget, and provides a flow diagram and a sequence of integration steps.
```

 Nutrition Facts                                                                          |                                                                                      |
| --------------------------------------------------------------------------------  | -------------------------------------------------------------------------               |

> | Learning outcomes                     | <ul><li>Understand the sequence of steps in the load widget flow.</li></ul>                                                       |

Reviewer: ```suggestion
| Learning outcomes                     | Understand the sequence of steps to load the Widget.                                                       |
```
